ui/app/components/ui/page-container/page-container-header/page-container-header.component.js

```diff
@@ -72,7 +73,7 @@ export default class PageContainerHeader extends Component {
 
         {
           onClose && headerCloseText
-            ? <div className="page-container__header-close-text" onClick={() => onClose()}>{ headerCloseText }</div>
+            ? <Button type="link" className="page-container__header-close-text" onClick={() => onClose()}>{ headerCloseText }</Button>
```

Comment:
Seems this changed the Close to move to a new line and to overlaid the Basic and Advanced Gas tabs, which Selenium has a failed on. 

<details>
<summary>Customize Gas Close modal text</summary>
<img src='https://user-images.githubusercontent.com/13376180/90548759-492d0600-e142-11ea-88df-d9d965bf11ad.png'>
<details>

Reply:
Good find, I fixed this issue for the choose recipients view on the send page, but missed it here. I'll apply the same fix there and get a revision up for you.